Bottom: 7e3e9d8c9ad24ca4a2fb68c4c23b4129dd6ac256
Top:    3db371792cd682d85f634f62a487c2a95b42ee6f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 14:52:17 -0500

Refresh of add-haproxy-role

---
diff --git a/roles/nexus3/defaults/main.yml b/roles/nexus3/defaults/main.yml
index e891ad741..ddc4d7015 100644
--- a/roles/nexus3/defaults/main.yml
+++ b/roles/nexus3/defaults/main.yml
@@ -17,9 +17,11 @@ nexus_container_name: 'nexus3'
 nexus_image_tag: 'nexus:{{ nexus_version}}'
 nexus_backup_volume: nexus-backup
 nexus_data_volume: nexus-data
-nexus_networks:
+nexus_networks: 
   - name: nexus3
-nexus_port_args: []
+nexus_port_args:
+  - "8081"
+  - "8443"
 nexus_volumes:
   - '{{ nexus_data_volume }}:{{ nexus_data_dir }}:z'
   - '{{ nexus_backup_volume }}:{{ nexus_backup_dir }}:z'
